include/libtorrent/alert_types.hpp

```diff
@@ -2864,6 +2865,9 @@ TORRENT_VERSION_NAMESPACE_2
 
 		std::string message() const override;
 
+		// id of the node the request was sent to (and this response was received from)
```

Comment:
as far as I can tell from the code, this comment is not correct. It's the ID that was passed to the `dht_sample_infohashes()` call.

Reply:
I notice I'm not using `node_id` consistently either...